docs/source/examples.md

```diff
@@ -274,3 +274,39 @@ By default, the `hi-ml` package assumes that you supply a commandline argument `
 the commandline) to trigger a submission of the present script to AzureML. If you wish to control it via a different
 flag, coming out of your own argument parser, use the `submit_to_azureml` argument of the function
 `health.azure.himl.submit_to_azure_if_needed`. 
+
+## Training with k-fold cross validation in Azure ML
+The example [sample 10](examples/10/sample.rst) demonstrates how to enable cross validation, by specifying the number
+of splits required.  If this number is > 1, a parent run is instantiated, with a number of child runs equalling the
+specified splits. For example,
```

Comment:
Maybe make that a bit more specific? There is a parent run with 5 child if doing 5-fold CV. First run gets additional arguments `--count=5 --index=0`, ...